src/harness/fourslashImpl.ts

```diff
@@ -822,6 +822,30 @@ export class TestState {
         });
     }
 
+    public baselineInlayHints(span: ts.TextSpan = { start: 0, length: this.activeFile.content.length }, preferences?: ts.UserPreferences): void{
+        interface HasPosition { position: number; }
+        const sortHints = (a: HasPosition, b: HasPosition) => {
+            return a.position - b.position;
+        };
+
+        const baselineFile = this.getBaselineFileNameForContainingTestFile();
+        const hints = this.languageService.provideInlayHints(this.activeFile.fileName, span, preferences);
+        const annotations = ts.map(hints.sort(sortHints), hint => {
+            let annotation = "{\n\t";
+            annotation += [
+                `text: "${hint.text}"`,
+                `position: ${hint.position}`,
+                `kind: ${hint.kind}`,
+                `whitespaceBefore: ${hint.whitespaceBefore}`,
+                `whitespaceAfter: ${hint.whitespaceAfter}`,
+            ].join(",\n\t");
+            annotation += ",\n}\n";
+            return annotation;
+        });

```

Comment:
Any reason you can't just do

```suggestion
        annotations = JSON.stringify(hints.sort(sortHints));
```

Reply:
Because I want to format stuff lol